pistar-update

```diff
@@ -21,13 +21,16 @@ if ! flock -n 200 ; then
   exit 1
 fi
 
+gitRemote="nolith"
+gitBranch="M17"
```

Comment:
This makes things easier when experimenting with a pistar branch, but it must go back to origin/master before the merge.

```suggestion
gitRemote="origin"
gitBranch="master"
```